helix-view/src/editor.rs

```diff
@@ -233,6 +317,8 @@ impl Default for Config {
             cursor_shape: CursorShapeConfig::default(),
             true_color: false,
             search: SearchConfig::default(),
+            render_whitespace: Whitespace::Basic(WhitespaceValue::All),
```

Comment:
Might want to be

```suggestion
            render_whitespace: Whitespace::Basic(WhitespaceValue::None),
```

so this is disabled by default